edb/integration_test.go

```diff
@@ -883,10 +897,11 @@ type marbleServer struct {
 	rootCert       string
 	secretRootCert string
 	secretRootKey  string
+	manifest       string
 }
 
 func (m marbleServer) Activate(context.Context, *rpc.ActivationReq) (*rpc.ActivationResp, error) {
 	return &rpc.ActivationResp{Parameters: &rpc.Parameters{
-		Env: map[string]string{core.EnvAPIAddress: addrAPI, core.EnvDatabaseAddress: addrDB, core.EnvDataPath: m.dataDir, core.ERocksDBMasterKeyVar: "4142434445464748494a4b4c4d4e4f50", marble.MarbleEnvironmentRootCA: m.rootCert, db.EnvRootCertificate: m.secretRootCert, db.EnvRootKey: m.secretRootKey},
-	}}, nil
+		Env:   map[string]string{core.EnvAPIAddress: addrAPI, core.EnvDatabaseAddress: addrDB, core.EnvDataPath: m.dataDir, core.ERocksDBMasterKeyVar: "4142434445464748494a4b4c4d4e4f50", marble.MarbleEnvironmentRootCA: m.rootCert, db.EnvRootCertificate: m.secretRootCert, db.EnvRootKey: m.secretRootKey, core.EnvManifestFile: "/tmp/manifest.json"},
```

Comment:
```suggestion
		Env:  map[string]string{core.EnvAPIAddress: addrAPI, core.EnvDatabaseAddress: addrDB, core.EnvDataPath: m.dataDir, core.ERocksDBMasterKeyVar: "4142434445464748494a4b4c4d4e4f50", marble.MarbleEnvironmentRootCA: m.rootCert, db.EnvRootCertificate: m.secretRootCert, db.EnvRootKey: m.secretRootKey, core.EnvManifestFile: "/tmp/manifest.json"},
```

Reply:
Uh, why remove the whitespace there? It makes "Env" and "Files" look aligned and actually was created by VS Code's auto formatter.

Reply:
I see, never mind then.